src/GraphQL.Conventions/Adapters/Engine/GraphQLEngine.cs

```diff
@@ -261,10 +261,9 @@ public GraphQLExecutor NewExecutor(IRequestDeserializer requestDeserializer = nu
             return this;
         }
 
-        public string SerializeResult(ExecutionResult result)
-        {
-            return _documentWriter.WriteToStringAsync(result).GetAwaiter().GetResult();
-        }
+        public string SerializeResult(ExecutionResult result) => SerializeResultAsync(result).GetAwaiter().GetResult();
```

Comment:
Consider to remove sync API or make it obsolete.

Reply:
see #195